website/client/components/tasks/task.vue

```diff
@@ -417,12 +417,21 @@ export default {
       if (this.task.type === 'habit' && (this.task.up || this.task.down)) return true;
       return false;
     },
+    timeTillDue () {
+      const endOfToday = moment().endOf('day');
+      const endOfDueDate = moment(this.task.date).endOf('day');
+
+      return moment.duration(endOfDueDate.diff(endOfToday));
+    },
     isDueOverdue () {
-      return moment().diff(this.task.date, 'days') >= 0;
+      return this.timeTillDue.asDays() <= 0;
```

Comment:
I don't see how this works. You've defined a `timeTillDue` function, but you're not calling it as a function.

Shouldn't it be:

```
return this.timeTillDue().asDays() <= 0;
```

Reply:
@crookedneighbor this is a Vue computed property so it's accessed as a normal property not a function

Reply:
but @aquibm i don't think moment plays very well with Vue reactivity and since you're always using `timeTillDue.asDays()` I think you should rename `timeTillDue` to `daysTillDue` and just return a number from it calling `asDays` on the moment duration

Reply:
@paglias I need a `moment.duration()` in order to use moment's humanize function. If we change `timeTillDue` to days, we're going to have to convert it back into a duration on line `432` before being able to humanize it.

Reply:
Ah I see, I missed that line, in that case continue this way 👍